### PR TITLE
Fix #3359: Convert welcome.xml into single layout file pull request for issue

### LINKED ISSUE #3359: Convert welcome.xml into single layout file
Currently `welcome.xml` has 4 different qualifier files but instead we can convert this into one single file.

Make sure that it does not affect the UI for any screen. To check this correct take before and after screenshots of home screen for `mobile-portrait`, `mobile-landscape`, `tablet-portrait`, `tablet-landscape` and they should look exactly same as before.

### PR DESCRIPTION
**Pull request by TheSwarnim**
## Explanation

Fixes #3359
This PR Convert welcome.xml into a single layout file. In this PR welcome.xml files for different orientations such as land, sw600dp-land and sw600dp-port are deleted and all rendering of the screen is done by the main welcome.xml, and the margin & padding changes dynamically using dimens.xml accordingly.
Please note that to run the project one has to rebuild the project because deleting the orientations welcome.xml files some changes were taking place (Since I don't know this, I struggle for a long time) otherwise it will not work on tablet modes.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by anandwana001** [comment]
Hi @TheSwarnim 
Congratulations on your first Pull Request. 

Please follow the first two points from the checklist given in the description section.

**Comment by TheSwarnim** [comment]
> Hi @TheSwarnim
> Congratulations on your first Pull Request.
> 
> Please follow the first two points from the checklist given in the description section.

Thank you, sir, I'll continue to give contributions to this organization, and wants to learn from you in the coming days

**Comment by rt4914** [comment]
> > Hi @TheSwarnim
> > Congratulations on your first Pull Request.
> > Please follow the first two points from the checklist given in the description section.
> 
> Thank you, sir, I'll continue to give contributions to this organization, and wants to learn from you in the coming days

@TheSwarnim As per the issue description, please add `Before/After` screenshots of the HomeScreen in mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape so that we can easily compare if changes are done correctly or not. 

Also in practice whenever there is a UI change we always add screenshots to PRs and when there are test cases changes then we add screenshot of tests passing.

**Comment by TheSwarnim** [comment]
> > > Hi @TheSwarnim
> > > Congratulations on your first Pull Request.
> > > Please follow the first two points from the checklist given in the description section.
> > 
> > 
> > Thank you, sir, I'll continue to give contributions to this organization, and wants to learn from you in the coming days
> 
> @TheSwarnim As per the issue description, please add `Before/After` screenshots of the HomeScreen in mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape so that we can easily compare if changes are done correctly or not.
> 
> Also in practice whenever there is a UI change we always add screenshots to PRs and when there are test cases changes then we add screenshot of tests passing.

Ok sir
##Before
Screenshot_1624548825
Screenshot_1624548656
Screenshot_1624548660
Screenshot_1624548671
Screenshot_1624548678
Screenshot_1624548782
Screenshot_1624548787
Screenshot_1624548819

##After
Screenshot_1624549608
Screenshot_1624549612
Screenshot_1624549621
Screenshot_1624549625
Screenshot_1624549150
Screenshot_1624549154
Screenshot_1624549170
Screenshot_1624549175

**Comment by TheSwarnim** [comment]
> > > Hi @TheSwarnim
> > > Congratulations on your first Pull Request.
> > > Please follow the first two points from the checklist given in the description section.
> > 
> > 
> > Thank you, sir, I'll continue to give contributions to this organization, and wants to learn from you in the coming days
> 
> @TheSwarnim As per the issue description, please add `Before/After` screenshots of the HomeScreen in mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape so that we can easily compare if changes are done correctly or not.
> 
> Also in practice whenever there is a UI change we always add screenshots to PRs and when there are test cases changes then we add screenshot of tests passing.

Ok sir,
# Before
## mobile-portrait
Screenshot_1624548660
Screenshot_1624548656

## mobile-land
Screenshot_1624548678
Screenshot_1624548671

## tablet-port
Screenshot_1624548782
Screenshot_1624548787

## tablet-land
Screenshot_1624548825
Screenshot_1624548819

# After
## mobile-land
Screenshot_1624549150
Screenshot_1624549154

## mobile-port
Screenshot_1624549170
Screenshot_1624549175

## tablet-land
Screenshot_1624549608
Screenshot_1624549612

## tablet-port
Screenshot_1624549625
Screenshot_1624549621

File: app/src/main/res/layout/welcome.xml [hunk 1]
```diff
@@ -12,8 +12,8 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingStart="28dp"
-    android:paddingEnd="28dp">
+    android:paddingStart="@dimen/home_outer_margin"
+    android:paddingEnd="@dimen/home_outer_margin">
```
Comment (rt4914) [inline app/src/main/res/layout/welcome.xml]: Do not use already defined values for this as it created dependencies on one another.

Create new dimens values like `home_welcome_outer_padding`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@TheSwarnim Thanks. I have left one suggestion which applies to all files.

File: app/src/main/res/layout/welcome.xml (hunk 1 above)
Comment (TheSwarnim) [reply app/src/main/res/layout/welcome.xml]: Ok sir I'll see it and use different dimens.

**Comment by TheSwarnim** [comment]
Hello @rt4914, Sir, I've created the dimens values as per the requirement. Please review it

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/build/classes" />
-  </component>
-  <component name="ProjectType">
-    <option name="id" value="Android" />
-  </component>
-</project>
```
Comment (rt4914) [inline .idea/misc.xml]: Please revert these changes. This file should not be deleted. 
Once done, please open the `Files changed` section on your PR and make sure this file is not visible.

File: .idea/runConfigurations.xml [hunk 3]
```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rt4914) [inline .idea/runConfigurations.xml]: Please revert these changes. This file should not be deleted. 
Once done, please open the `Files changed` section on your PR and make sure this file is not visible.

File: build.gradle [hunk 4]
```diff
@@ -1,31 +0,0 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
```
Comment (rt4914) [inline build.gradle]: Please revert these changes. This file should not be deleted. 
Once done, please open the `Files changed` section on your PR and make sure this file is not visible.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested changes for files which should not be changed. Otherwise everything looks good.

**Comment by TheSwarnim** [comment]
Hello @rt4914, Sir, please review it again, the error has occurred because I've not added these above files in commit mistakingly.

File: build.gradle (hunk 4 above)
Comment (rt4914) [reply build.gradle]: > Hello @rt4914, Sir, please review it again, the error has occurred because I've not added these above files in commit mistakingly.

@TheSwarnim Thanks. Also make sure that you reply to all open comments so that the reviewer can understand what was the step taken on that particular comment.

**Comment by rt4914** [comment]
> Hello @rt4914, Sir, please review it again, the error has occurred because I've not added these above files in commit mistakingly.

@TheSwarnim Thanks. Also make sure that you reply to all open comments so that the reviewer can understand what was the step taken on that particular comment.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by rt4914** [comment]
@TheSwarnim Please update your branch by merging with latest develop so that we can merge this PR
